src/nvim/ex_cmds.c

```diff
@@ -4822,8 +4822,8 @@ static void helptags_one(char_u *dir, char_u *ext, char_u *tagfname,
 
   // Find all *.txt files.
   size_t dirlen = STRLEN(dir);
-  STRCPY(NameBuff, dir);
-  STRCAT(NameBuff, "/**/*");
+  STRLCPY(NameBuff, dir, sizeof(NameBuff));
+  STRCAT(NameBuff, "/**/*");  // NOLINT
```

Comment:
For some reason I couldn't get `NOLINT(readability/old_style_comment)` to work here. `clint` complains that it's an unknown category, but reports the same. Tried variations, including just `readability`, just `old_style_comment`, `readability/old-style-comment`. No luck. 

```
src/nvim/ex_cmds.c:4826:  Unknown NOLINT error category: readability/old_style_comment  [readability/nolint] [5]
src/nvim/ex_cmds.c:4826:  /*-style comment found, it should be replaced with //-style.  /*-style comments are only allowed inside macros.  Note that you should not use /*-style comments to document macros itself, use doxygen-style comments for this.  [readability/old_style_comment] [5]
Total errors found: 2
```